CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 6.2.1
```

Comment:
Should be rather 6.2.0 :wink:

Reply:
Whoops, deleted branch and created new PR:

https://github.com/nextcloud/richdocuments/pull/2360